global.d.ts

```diff
@@ -0,0 +1,21 @@
+declare interface BosContext {
+  accountId?: string;
+  networkId: NetworkId;
+  props: object;
+}
+
+declare const Widget: (params: {
+  src: string;
+  props: object;
+}) => React.ReactNode;
+
+declare interface BaseProps {
+  page: any;
+}
```

Comment:
This looks like platform BaseProps, but these are DevHub BaseProps. Consider renaming it, adding a comment, or use explicit import instead of relying implicitly on this type.